aim2dat/strct/strct.py

```diff
@@ -621,10 +621,43 @@ def from_aiida_structuredata(
     @export_method
     def to_file(self, file_path: str) -> None:
         """
-        Export structure to file using the ase interface.
+        Export structure to file using the ase interface or certain file formats for Zeo++.
         """
-        backend_module = _return_ext_interface_modules("ase_atoms")
-        backend_module._write_structure_to_file(self, file_path)
+        if file_path.endswith(".cssr"):
+            f = open(file_path, "w")
+            f.write(" ".join(map(str, self.cell_lengths)) + "\n")
+            f.write(" ".join(map(str, self.cell_angles)))
+            f.write(" SPGR = " + self.determine_space_group()["space_group"]["int_symbol"] + "\n")
+            f.write(f"{len(self.positions)} 0\n")
+            f.write(f"0 {self.label}\n")
+            for i, el_pos in enumerate(zip(self.elements, self.scaled_positions)):
+                f.write(
+                    f"  {i+1} {el_pos[0]} {el_pos[1][0]} {el_pos[1][1]} {el_pos[1][2]} "
+                    + " ".join(map(str, 9 * [0]))
+                    + "\n"
+                )
+            f.close()
```

Comment:
I only mention it here but this would be valid for all the different file formats. Instead of explicitly opening and closing the file for each file format, you could append all the different lines for the input file to a list, e.g. `output`. Afterwards, you could use something like

```
with open(file_path, 'w') as file:
    file.write('\n'.join(output))
```
to finally write the file.

Reply:
Thanks for the suggestion. I will fix it.